crates/ruff_python_formatter/src/builders.rs

```diff
@@ -219,7 +219,6 @@ impl<'fmt, 'ast, 'buf> JoinCommaSeparatedBuilder<'fmt, 'ast, 'buf> {
             if let Some(last_end) = self.entries.position() {
                 let magic_trailing_comma = has_magic_trailing_comma(
                     TextRange::new(last_end, self.sequence_end),
-                    self.fmt.options(),
```

Comment:
I noticed that we pass both options and `context`. This is unnecessary because the options can be retrieved from `context.options()`